src/main/kotlin/no/nav/k9/general/systemauth/AccessTokenClientResolver.kt

```diff
@@ -10,31 +10,34 @@ import org.slf4j.Logger
 import org.slf4j.LoggerFactory
 
 internal class AccessTokenClientResolver(
-    clients: Map<String, Client>
+    private val clients: Map<String, Client>
 ) {
     companion object {
         private val logger: Logger = LoggerFactory.getLogger(AccessTokenClientResolver::class.java)
         private const val AZURE_V2_ALIAS = "azure-v2"
+        private const val TOKEN_X_ALIAS = "tokenx"
     }
 
-    private val azureV2Client = clients.getOrElse(AZURE_V2_ALIAS) {
-        throw IllegalStateException("Client[$AZURE_V2_ALIAS] må være satt opp.")
-    } as PrivateKeyClient
+    internal val tokenxClient get() = createSignedJwtAccessTokenClient(resolveClient(TOKEN_X_ALIAS))
```

Comment:
Er det noen grunn til å bruke eksplisitt getter her og under?

Reply:
Ja, denne måten brukes for å aksessere et felt med get. 
Basically, man henter verdien av en funksjon som property. 
Akkurat i det tilfelle der virker det rart fordi den funksjonen gjør ikke annet enn å kalle createSignedJwtAccessTokenClient().

Et bruksområde kunne ha vært e.g. `val sum get() = x + y`

Reply:
Men er det nødvendig? Kan man ikke bare skrive `val tokenxClient = createSignedJwtAccessTokenClient(resolveClient(TOKEN_X_ALIAS))`

Reply:
Det stemmer at den er unødvendig. Will fix 👍